pom.xml

```diff
@@ -53,11 +53,11 @@
         <mockito-junit-jupiter.version>5.2.0</mockito-junit-jupiter.version>
         <mockserver-client-java.version>5.15.0</mockserver-client-java.version>
         <nimbus-jose-jwt.version>9.40</nimbus-jose-jwt.version>
-        <private-api-sdk-java.version>4.0.168</private-api-sdk-java.version>
+        <private-api-sdk-java.version>4.0.288</private-api-sdk-java.version>
         <sdk-manager-java.version>3.0.9</sdk-manager-java.version>
         <snakeyaml.version>2.2</snakeyaml.version>
         <snappy-java.version>1.1.10.5</snappy-java.version>
-        <spring-boot.version>3.3.5</spring-boot.version>
+        <spring-boot.version>3.4.4</spring-boot.version>
```

Comment:
When trying to test the service in docker there is a not found error coming back from this service as it can no longer find the maintenance endpoint.
Is seems to be due to the upgrade to Spring 3.4.4. Some of the actuator properties have been deprecated.
You will need to update the application.properties file replace
management.endpoints.enabled-by-default=${MANAGEMENT_ENDPOINTS_ENABLED_BY_DEFAULT}

with 
management.endpoints.enabled=${MANAGEMENT_ENDPOINTS_ENABLED_BY_DEFAULT}
That should resolve the issue.